config/locales/es.yml

```diff
@@ -110,6 +110,9 @@ es:
         updated_not_active: "Su contraseña ha sido restablecida, pero su correo electrónico aún no ha sido confirmado."
         updated: "Su contraseña ha sido cambiada con éxito. Ya tienes la sesión iniciada."
         send_instructions: "Recibirás un correo electrónico con instrucciones sobre cómo confirmar su cuenta en unos minutos."
+    mailer:
```

Comment:
we should remove this one and the keys below. As @luisramos0 explained, that's exactly what translators will do in Transifex based on config/locales/en.yml.